picard/formats/id3.py

```diff
@@ -143,6 +148,7 @@ class ID3File(File):
         'MusicBrainz Release Track Id': 'musicbrainz_trackid',
         'MusicBrainz Disc Id': 'musicbrainz_discid',
         'MusicBrainz Work Id': 'musicbrainz_workid',
+        'MusicBrainz Movement Work Id': 'musicbrainz_movementid',
```

Comment:
IMO yes. A plugin to replicate the iTunes functionality from MB data would move the Picard Work field to Movement so it needs a Work Id field to match,.